lib/promscrape/discovery/gce/gce_test.go

```diff
@@ -0,0 +1,27 @@
+package gce
```

Comment:
Is it fine to create a new `gce_test.go` for testing this?

Previously `gce.go` doesn't have any related test files.